docs/docs/cmd/spo/site/site-apppermission-add.md

```diff
@@ -0,0 +1,36 @@
+# spo site apppermission add
+
+Adds a specific application permissions to the site
+
+## Usage
+
+```sh
+m365 spo site apppermission add [options]
+```
+
+## Options
+
+`-u, --siteUrl <siteUrl>`
+: URL of the site collection to add the permission
+
+`-p, --permission <permission>`
+: Permission to site (`read`, `write`, `read,write`). If multiple permissions have to be granted, they have to be comma separated ex. `read,write`
+
+`-i, --appId <appId>`
+: App Id
+
+`-n, --appDisplayName <appDisplayName>`
+: App display name
+
+--8<-- "docs/cmd/_global.md"
+
+## Remarks
+Pass in appId and/or addDisplayName, use both for best performance avoiding the extra lookup.
```

Comment:
```suggestion

To set permissions, specify at minimum either `appId` or `addDisplayName`. For best performance specify both values to avoid extra lookup.
```